app/models/service_retire_task.rb

```diff
@@ -41,23 +40,25 @@ def create_retire_subtasks(parent_service)
       nh['options'] = options.except(:child_tasks)
       # Initial Options[:dialog] to an empty hash so we do not pass down dialog values to child services tasks
       nh['options'][:dialog] = {}
-      new_task = create_task(svc_rsc, parent_service, nh)
+      new_task = create_task(svc_rsc, parent_service, nh, parent_task)
```

Comment:
@d-m-u The `nh` has is pulling in virtual columns which is causing the test failure:
```
Failure/Error: new_task = (svc_rsc.resource.type.demodulize + "RetireTask").constantize.new(nh)
ActiveModel::MissingAttributeError:
  can't write unknown attribute `href_slug`
# ./app/models/service_retire_task.rb:53:in `create_task'
# ./app/models/service_retire_task.rb:43:in `block in create_retire_subtasks'
...
```

You can fix this by excluding the virtual columns on line 39 using `self.class.virtual_attribute_names`
```ruby
nh = attributes.except("id", "created_on", "updated_on", "type", "state", "status", "message", *self.class.virtual_attribute_names)
```